components/hybrid_engine/src/snapshot.rs

```diff
@@ -165,15 +168,17 @@ mod tests {
             assert!(!iter.seek_to_first().unwrap());
         }
         let mut write_batch = hybrid_engine.write_batch();
-        write_batch.prepare_for_range(range.clone());
+        write_batch.prepare_for_region(&region);
         write_batch
             .cache_write_batch
             .set_range_cache_status(RangeCacheStatus::Cached);
-        write_batch.put(b"hello", b"world").unwrap();
+        write_batch.put(b"zhello", b"world").unwrap();
```

Comment:
Out of curiosity, why prefix "z"?

Reply:
"z" is actually the `DATA_KEY_PREFIX`. Because after this change, the data range is built from region. Thus a region with range ["k00", "k10") actually covers the data key range ["zk00", "zk10")